main.tf

```diff
@@ -118,18 +118,20 @@ locals {
   cluster_master_auth_map         = "${local.cluster_master_auth_list_layer2[0]}"
 
   # cluster locals
-  cluster_name                = "${local.cluster_type_output_name[local.cluster_type]}"
-  cluster_location            = "${local.cluster_type_output_location[local.cluster_type]}"
-  cluster_region              = "${local.cluster_type_output_region[local.cluster_type]}"
-  cluster_zones               = "${sort(local.cluster_type_output_zones[local.cluster_type])}"
-  cluster_endpoint            = "${local.cluster_type_output_endpoint[local.cluster_type]}"
-  cluster_ca_certificate      = "${lookup(local.cluster_master_auth_map, "cluster_ca_certificate")}"
-  cluster_master_version      = "${local.cluster_type_output_master_version[local.cluster_type]}"
-  cluster_min_master_version  = "${local.cluster_type_output_min_master_version[local.cluster_type]}"
-  cluster_logging_service     = "${local.cluster_type_output_logging_service[local.cluster_type]}"
-  cluster_monitoring_service  = "${local.cluster_type_output_monitoring_service[local.cluster_type]}"
-  cluster_node_pools_names    = "${local.cluster_type_output_node_pools_names[local.cluster_type]}"
-  cluster_node_pools_versions = "${local.cluster_type_output_node_pools_versions[local.cluster_type]}"
+  cluster_name                   = "${local.cluster_type_output_name[local.cluster_type]}"
+  cluster_location               = "${local.cluster_type_output_location[local.cluster_type]}"
+  cluster_region                 = "${local.cluster_type_output_region[local.cluster_type]}"
+  cluster_zones                  = "${sort(local.cluster_type_output_zones[local.cluster_type])}"
+  cluster_endpoint               = "${local.cluster_type_output_endpoint[local.cluster_type]}"
+  cluster_ca_certificate         = "${lookup(local.cluster_master_auth_map, "cluster_ca_certificate")}"
+  cluster_master_version         = "${local.cluster_type_output_master_version[local.cluster_type]}"
+  cluster_min_master_version     = "${local.cluster_type_output_min_master_version[local.cluster_type]}"
+  cluster_logging_service        = "${local.cluster_type_output_logging_service[local.cluster_type]}"
+  cluster_monitoring_service     = "${local.cluster_type_output_monitoring_service[local.cluster_type]}"
+  cluster_private_cluster        = "${local.cluster_type_output_private_cluster[local.cluster_type]}"
```

Comment:
It looks like you're missing the actual creation of the `cluster_type_output_private_cluster` and `cluster_master_ipv4_cidr_block ` maps which this references.